src/main/spec/internal-tally-api-spec.yaml

```diff
@@ -24,6 +24,40 @@ servers:
         default: rhsm-subscriptions
 
 paths:
+  /v1/instances/billing_account_ids:
```

Comment:
If this endpoint is meant to be used by the UI, the endpoint definition should be in https://github.com/RedHatInsights/rhsm-subscriptions/blob/main/api/rhsm-subscriptions-api-v1-spec.yaml.

This API definition already contains security handling: https://github.com/RedHatInsights/rhsm-subscriptions/blob/37907a07760179e1d450bf48432a3cdf5996919b/api/rhsm-subscriptions-api-v1-spec.yaml#L672

Reply:
I think we're trying to move away from having all the apis in swatch-api. The card says:

Configuration:
Add an entry to the nginx reverse proxy configuration in the swatch-api to direct requests to /api/rhsm-subscriptions/v1/instances/billing_account_ids to /api/swatch-tally/v1/instances/billing_account_ids

Reply:
I have this WIP here: https://github.com/RedHatInsights/rhsm-subscriptions/pull/4215/files

Reply:
"swatch-tally" is the "worker" profile in the monolith which sounds odd to me.
Not sure whether moving away from having all the apis in swatch-api is entirely correct: I thought this was only applicable for the APIs that has been moved from the monolith to a concrete service like it's your case (the subscription API got moved to the swatch-contracts)
@lindseyburnett can you clarify this?

Reply:
We had a call and came to the same conclusion as above :) It's good as is in the monolith.